transforms/universal/fdedup/python/pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[project]
+name = "dpk_fdedup_transform_python"
+version = "0.2.2.dev1"
+requires-python = ">=3.10"
+description = "Fuzzy Dedup Transform for Python"
+license = {text = "Apache-2.0"}
+readme = {file = "README.md", content-type = "text/markdown"}
+authors = [
+    { name = "Nelson Bore", email = "k.nelsonbore@gmail.com" },
+    { name = "Constantin Adam", email = "cmadam@us.ibm.com" },
+]
+dependencies = [
+    "data-prep-toolkit==0.2.2.dev1",
```

Comment:
Current version is dev2

Reply:
Fixed in commit https://github.com/IBM/data-prep-kit/pull/699/commits/2f61be7938d7540a0a1831e85b8a961bef24d35c